rocksdb_admin/helix_client.cpp

```diff
@@ -34,7 +34,7 @@ JNIEnv* createVM(const std::string& class_path) {
   JavaVM* jvm;
   JNIEnv* env;
   JavaVMInitArgs args;
-  JavaVMOption options[2];
+  JavaVMOption options[3];
 
   args.version = JNI_VERSION_1_6;
   args.nOptions = 2;
```

Comment:
this needs to be changed?

Reply:
good catch... I tested this by editing the submodule in cosmos and forgot to copy this part over